### ISSUE TITLE
Padding above Tabs is Back in FF v108

### ISSUE THREAD

**Issue by gashtal** [Aris-t2/CustomCSSforFx#528]
The same issue as [this issue](https://github.com/Aris-t2/CustomCSSforFx/issues/471) is again back in FF v108, preventing tabs from being selected, or closed with middle-click, if the mouse cursor is at the very top of the screen.

1. Does the issue occur without any files inside /chrome/ folder?
N/A

2. Issue description, userChrome.css/userContent.css 'settings' the issue occurs with and steps to reproduce the issue:
Settings file: [userChrome.zip](https://github.com/Aris-t2/CustomCSSforFx/files/10224104/userChrome.zip)

3. Does the issue occur with the most recent version of CustomCSSforFx (release/test/dev build)?
Yes, reproduced with latest v4.2.6.

4. Screenshots showing the issue (drag & drop images into this post):
![image](https://user-images.githubusercontent.com/27990818/207501461-c638af1f-4a16-4b02-a014-eb067ad9bb38.png)

5. System information
- OS & OS version: **Windows 10 22H2 x64**
- DPI / HiDPI (e.g. 100%, 125%...): **100%**
- Firefox version: **108.0 (64-bit)**
- Firefox theme (e.g. default, build in light/dark, external theme + link): **default**
- Is 'about:config > toolkit.legacyUserProfileCustomizations.stylesheets' set to 'true': **Yes**
- Is 'about:config > layout.css.moz-box-flexbox-emulation.enabled' set to 'true' (Fx 107 only): **N/A**

**Comment by Aris-t2** [comment]
Its a tab related issue inside classic_squared_tabs.css and classic_squared_tabs_australized.css files.

This should fix the issue:
```css
#main-window[sizemode="maximized"][tabsintitlebar] #TabsToolbar {
  margin-top: -3px !important;
}
```

**Comment by Aris-t2** [comment]
The fix is part of [4.2.7](https://github.com/Aris-t2/CustomCSSforFx/releases) release now.
